src/analysis/processing/qgsalgorithmdetectdatasetchanges.cpp

```diff
@@ -0,0 +1,432 @@
+/***************************************************************************
+                         qgsalgorithmdetectdatasetchanges.cpp
+                         -----------------------------------------
+    begin                : December 2019
```

Comment:
January 2020

Reply:
Well, I started this last year so technically it did begin in December!